### ISSUE TITLE
[A11Y] Unable to navigate using right/left keys on “What is a web compatibility issue” popup

### ISSUE THREAD

**Issue by softvision-oana-arbuzov** [webcompat/webcompat.com#3417]
**Browser / Version**: Firefox Nightly 81.0a1 (2020-07-27) 
**Operating System**: Windows 10 Pro

**Steps to Reproduce**:
1. Navigate to https://webcompat.com/issues/new
2. Using "Tab" key, navigate to the URL field and fill it in. 
3. Using "Tab" key, navigate to "Confirm" button.
4. Using "Tab" key, navigate to "Learn more about web compatibility" link.
5. Using "Enter" key, activate the popup.
6. Using "Right" key, try to navigate to the next slide.
7. Observe behavior.

**Expected Behavior:**
The next slide is displayed.

**Actual Behavior:**
Nothing happens, the next slide is not displayed.

**Notes:**
1. Screenshot attached.
2. According to "Inspector>Accessibility" tab, the navigation dots "must be focusable and should have interactive semantics".
![image](https://user-images.githubusercontent.com/12184325/88664116-3ac07100-d0e5-11ea-8619-326978ce1ad1.png)

![image](https://user-images.githubusercontent.com/12184325/88663609-8e7e8a80-d0e4-11ea-8b15-a6a8af178b6a.png)

Watchers:
@softvision-oana-arbuzov
@cipriansv

**Comment by softvision-oana-arbuzov** [comment]
I can confirm this is fixed.

**Tested with:**
Browser / Version: Firefox Nightly 82.0a1 (2020-09-16), Chrome 85.0.4183.102
Operating System: Windows 10 Pro
